src/aui/framemanager.cpp

```diff
@@ -647,7 +648,11 @@ wxAuiManager::~wxAuiManager()
 
     delete m_art;
 }
-
+void wxAuiManager::OnSysColourChanged(wxSysColourChangedEvent& event) {
+    m_art->SysColourChanged();
+    m_frame->Refresh();
```

Comment:
It seems a bit weird to have to refresh the frame from this handler. Shouldn't it be refreshed from its own handler?

Reply:
I had a strange race condition where the frame would gets it's wxSysColourChangedEvent and refresh itself prior to the AuiManager getting its event so it wasn't picking up the new colors. That said, I cannot reproduce it right now.

Reply:
No, the m_frame needs to be fully refreshed.   Otherwise none of the docked panels get updated and they continue to display their old colors.  The other option might be to go and find all the docked panels and windows and refresh each one individually, but the full refresh seems to work best.